File: frontend/src/app/models/Payment.ts
export class Payment {
    interestAmount: number;
    payAmount: number;
    totalAmount: number;
    monthNumber: number;
    remainingAfter: number;


    constructor(object: any = null) {
        if (object !== null) {
            this.interestAmount = object.interestAmount;
            this.payAmount = object.payAmount;
            this.totalAmount = object.totalAmount;
            this.monthNumber = object.monthNumber;
            this.remainingAfter = object.remainingAfter;
        }
    }
}

File: backend/LoanThingy.Models2/Payment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LoanThingy.Models
{
    public class Payment
    {
        public decimal interestAmount { get; set; }
        public decimal payAmount { get; set; }
        public decimal totalAmount { get; set; }
        public int monthNumber { get; set; }
        public decimal remainingAfter { get; set; }

        public Payment() { }
        public Payment(decimal interestAmount,
            decimal payAmount,
            decimal totalAmount,
            int monthNumber)
        {
            this.interestAmount = interestAmount;
            this.payAmount = payAmount;
            this.totalAmount = totalAmount;
            this.monthNumber = monthNumber;
        }
    }
}
